.github/workflows/pr.yaml

```diff
@@ -118,10 +119,18 @@ jobs:
       arch: "amd64"
       container_image: "rapidsai/ci-conda:latest"
       run_script: "ci/build_docs.sh"
-  wheel-build-cudf:
+  wheel-build-libcudf:
```

Comment:
Like with the script names below, let's start matching our conda conventions here and use `wheel-build-cpp` and `wheel-build-python` for names.

Reply:
This thread seems out-of-date now that pylibcudf exists. Are there any actionable pieces remaining?

Reply:
I should have added another comment. I revived this as the partner of this one: https://github.com/rapidsai/cudf/pull/15483#discussion_r1725527992

Reply:
Let's discuss in the other thread.

Reply:
that other thread concluded with "this can happen in a follow-up", so I'll resolve this.